### PR TITLE
[improve] : Optimize queues to obtain data and prevent cpu idling

### PR DESCRIPTION
**Pull request by Yanshuming1**
Optimize queues to obtain data and prevent cpu idling

### CODE REVIEW

File: manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/DispatcherAlarm.java [hunk 1]
```diff
@@ -135,6 +135,7 @@ public void run() {
                     }
                 } catch (IgnoreException ignored) {
                 } catch (InterruptedException e) {
+                    Thread.currentThread().interrupt();
```
Comment (yuluo-yx) [inline manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/DispatcherAlarm.java:138]: this code should be in the previous catch block. right?

Comment (Yanshuming1) [reply manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/DispatcherAlarm.java:138]: Code block in InterruptedException

Comment (yuluo-yx) [reply manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/DispatcherAlarm.java:138]: got it.

**Review by tomsun28 — APPROVED** [review]
lgtm!👍

**Pull request merged**